UI/data/themes/Yami_Grey.ovt

```diff
@@ -0,0 +1,18 @@
+@OBSThemeMeta {
+    name: 'Grey';
+    id: 'com.obsproject.Yami.Grey';
```

Comment:
I prefer "gray", but consider this a dismissable suggestion rather than a blocker.